Makefile

```diff
@@ -17,6 +17,11 @@ $(HEXBIN_OUT): $(HEXBIN_IN)
 
 %.ipynb.raw.html: %.ipynb
 	jupyter-nbconvert $< --output $(notdir $@) --to html --template basic
+	if [ "results/"  = "$(dir $@)" ]; then \
+	  echo 'got here'; \
```

Comment:
If this is meant to stay, please replace "got here" with a meaningful message, perhaps including the calling process and the filename.

Reply:
Removed it.